### PR TITLE
Make `downloaded_media` path consistent between Windows and Linux

### LINKED ISSUE #22: [Enhancement] Use the same path for EmulationStation DE metadata on Linux and Windows
Currently EmulationStation DE for Windows uses the path `Emulation\storage\downloaded_media\` under the configured EmuDeck drive. EmulationStation DE on SteamDeck uses `Emulation\tools\downloaded_media\`. Neither is the default ESDE path for metadata. Using the same path on both platforms allows sharing metadata when cloning or synching EmuDeck data.

### PR DESCRIPTION
**Pull request by shawntabrizi**
Closes #22 

This PR updates the paths, scripts, and configuration files which place the Emulation Station `downloaded_media` into the `tools/downloaded_media` folder which is consistent with the Linux / Steam Deck configuration.

### CODE REVIEW

File: configs/emulationstation/es_settings.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <string name="ROMDirectory" value="/run/media/mmcblk0p1/Emulation/roms/" />
-<string name="MediaDirectory" value="/run/media/mmcblk0p1/Emulation/storage/download_media" />
```
Comment (shawntabrizi) [inline configs/emulationstation/es_settings.xml:3]: Independent of this PR being accepted, I believe there is a typo here of `downloaded_media`

**Comment by shawntabrizi** [comment]
Note that not only does this PR being consistency between Linux and Windows, but it seems there were other places in the code base which did already assume the files were in the `tools` folder. For example:

https://github.com/EmuDeck/emudeck-we/blob/2a6f9136e73f93c5b76cf3952ece36d36570e0b5/functions/ToolScripts/emuDeckInstallHomebrewGames.ps1#L9

**Comment by dragoonDorise** [comment]
Hi @shawntabrizi , sorry I missed this. I've fixed some conflicts but this PR need to have into account old users with the old path, so this also should migrate old users to the new paths so if they do a reset config don't "lose" all their artwork because the directory changed

**Comment by shawntabrizi** [comment]
@dragoonDorise Thanks for getting back to me.

I don't think I will complete this PR at this moment.

I think you might have already solved these problems anyway?

In any case, thanks for your time and review.

**Pull request closed without merging**